### PR TITLE
makefile integration target to submit all the integration payloads

### PR DESCRIPTION
**Pull request by jhkennedy**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -12,6 +13,10 @@ test_file ?= 'tests/'
 test:
 	pytest $(test_file)
 
+integration:
+	export AWS_PAGER='' && \
+	$(foreach file, $(wildcard tests/integration/*.json), echo "aws sns publish --profile saml-pub --topic-arn ${TEST_TOPIC_ARN} --message file://${file} --output json";)
```
Comment (jtherrmann) [inline Makefile:18]: Probably hitting the limits of my bash knowledge, but why is there an `echo` before the `aws sns` command?

Comment (jhkennedy) [reply Makefile:18]: because I was testing it without running it and accidentelly committed the echo :rofl:

Comment (jhkennedy) [reply Makefile]: I should just be using the `--just-print` or `-n` make options:
https://web.mit.edu/gnu/doc/html/make_5.html

Comment (jtherrmann) [reply Makefile]: Nice, did not know about those options.

**Review by jtherrmann — APPROVED** [review]

**Pull request merged**